src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -10,6 +10,70 @@
 
 namespace DSCPython
 {
+    /// <summary>
+    /// This class wraps a PythonNet.PyObj and performs
+    /// disposal tasks to make sure the underlying object is removed from
+    /// the shared global scope between all CPython scopes.
+    /// </summary>
+    [IsVisibleInDynamoLibrary(false)]
+    public class DynamoCPythonHandle : IDisposable
```

Comment:
Ok. Now I see we need to do some special handling for disposal, so using plain `PyObject` won't do.

Anyway, this is a secondary discussion 🙂

Reply:
Does this class need to be imported into the VM?

Reply:
yes it does.